packages/new-deploy/src/bridge/BridgeContracts.ts

```diff
@@ -170,14 +184,25 @@ export default class BridgeContracts extends AbstractBridgeDeploy<config.EvmBrid
     const ubc = core.upgradeBeaconController.address;
     if (!ubc) throw new Error('Cannot deploy proxy without UBC');
     const factory = new UpgradeBeacon__factory(this.context.getDeployer(name));
-    const beacon = await factory.deploy(implementation, ubc, this.overrides);
+
+    const constructorArguments = [implementation, ubc];
+    const beacon = await factory.deploy(
+      constructorArguments[0],
```

Comment:
could do `...constructorArguments` so the code is the same no matter how many args there are

Reply:
doesn't work. spread operator only works for rest args :(